### ISSUE TITLE
Difficulties for interacting with the image block when is in a circle form with a small size

### ISSUE THREAD

**Issue by jesusyesares** [WordPress/gutenberg#63254]
### Description
Trying one of the patterns in Twenty-Twenty-Four Theme, I tried to change the image of a cite. This image is inside a circle form in a group block. Is something like this:

![CleanShot 2024-07-08 at 13 38 43@2x](https://github.com/WordPress/gutenberg/assets/2352123/0d426c06-e019-48fa-a6eb-d7537a4416dd)

The properties of the block are something like this:

![CleanShot 2024-07-08 at 13 39 18@2x](https://github.com/WordPress/gutenberg/assets/2352123/b5e3af83-3a41-42bc-b2c4-8c91bdc3e94c)

The problem is that there's no access to the upload button or the Media library. If you want to access to this buttons, you have to increase the width and the height or change the aspect ratio to Original as you can see here.

![CleanShot 2024-07-08 at 13 40 57@2x](https://github.com/WordPress/gutenberg/assets/2352123/a15d5306-ae5d-473b-9b5c-7869b69a336c)

![CleanShot 2024-07-08 at 13 41 35@2x](https://github.com/WordPress/gutenberg/assets/2352123/c5c5c76b-c85e-4e0d-9adf-01cb70b9a64b)

Or you can do this:

![CleanShot 2024-07-08 at 13 41 27@2x](https://github.com/WordPress/gutenberg/assets/2352123/fa3e3f8b-4bd2-4731-a46b-7ae763359f66)

![CleanShot 2024-07-08 at 13 41 04@2x](https://github.com/WordPress/gutenberg/assets/2352123/43c65825-1cf4-4d80-893d-102e9a75b975)

### Steps to Reproduce
1. In the Twenty Twenty Four Theme.
2. In the home page.
3. Go to the Testimonial Block.

![CleanShot 2024-07-08 at 16 46 42@2x](https://github.com/WordPress/gutenberg/assets/2352123/e7fd70d3-5783-4f71-879d-b1c0288c7386)

4. 🐞 Bug occurs.

### Expected Results
1.  ✅ At least it must be possible to access to the images from the lateral panel under the block attributes.

![CleanShot 2024-07-08 at 13 41 04@2x](https://github.com/WordPress/gutenberg/assets/2352123/cff84597-3c52-4d68-9a09-1986ffd55b12)

### Actual Results
1.  ❌ What actually happened.

![CleanShot 2024-07-08 at 13 38 43@2x](https://github.com/WordPress/gutenberg/assets/2352123/a76b37b4-0166-4939-a288-d58e7e36bd51)

![CleanShot 2024-07-08 at 13 40 57@2x](https://github.com/WordPress/gutenberg/assets/2352123/f2d52800-ba06-49b0-b276-0af0b95899b1)

### Environment
- WordPress: 6.6-RC2
- PHP: 8.2.20
- Server: Apache
- Database: mysqli (Server: 10.11.8-MariaDB-ubu2004-log / Client: mysqlnd 8.2.20)
- Browser: Chrome 126.0.0.0 (macOS)
- Theme: Twenty Twenty-Four 1.1
- MU-Plugins: 
  * ManageWP - Worker Loader 1.0.0
- Plugins: 
  * Create Block Theme 2.3.0
  * Gutenberg 18.7.1
  * WordPress Beta Tester 3.5.6

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by jasmussen** [comment]
Good issue. This is something we need to address, and I just referred to this issue from #63327 which details the Site Logo placeholder state which actually addresses the issue at hand here, but could be improved with a better tooltip placement and controls in the inspector.

The Site Logo presents a path forward for how to address this issue. I'd love to hear from @WordPress/gutenberg-design if they agree. Depending on discussion I would then think both of these issues can be fixed together.

**Comment by jameskoster** [comment]
I agree in general. The Featured Image block is the same (when page editing) and works quite well:

<img width="1356" alt="Screenshot 2024-07-10 at 11 47 42" src="https://github.com/WordPress/gutenberg/assets/846565/48557d6b-aa66-4bbf-ab3a-b0c6d28264a5">


But these block's on-canvas UX are missing a way to 'Insert from URL'. In the case of regular images, maybe the Upload button should open a menu with options: "Upload", "Media library", "Insert from URL"?

**Comment by jasmussen** [comment]
This issue will be fixed by #64320.

**Comment by Mamaduka** [comment]
Can we close this now that #64320 is merged? There's also a general issue for tracking placeholder improvements - #64288.

**Comment by t-hamano** [comment]
I think #64288 covers this issue so I think we can close this issue.
